src/components/DriverOverview/DriverOverview.tsx

```diff
@@ -0,0 +1,208 @@
+"use client";
+
+import React from "react";
+import { Text, Document, Page, View, StyleSheet, Image } from "@react-pdf/renderer";
+
+export interface DriverOverviewCardProps {
+    driverName: string;
+    date: Date;
+    data: DriverOverviewTableData[];
+}
+
+export interface DriverOverviewTableData {
+    name: string;
+    address: {
+        line1: string;
+        line2: string;
+        town: string;
+        county: string;
+        postcode: string;
+    };
+    contact?: string;
+    packingDate: Date;
+    parcels: number | "None";
+    instructions?: string;
+}
+
+const styles = StyleSheet.create({
+    container: {
+        padding: 25,
+        alignItems: "center",
+    },
+    infoAndLogoContainer: {
+        height: "100",
```

Comment:
%? units? can this just be a number if react-pdf does it for you?

Reply:
Yes this is pt, the default setting for react-pdf

Reply:
should this just be a number (not a string)?